portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -0,0 +1,25 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/form-handler")
+public class FormHandlerServlet extends HttpServlet {
+
+  @Override
+  public void doPost(final HttpServletRequest request, final HttpServletResponse response) throws IOException {
+
+    // Get the value entered in the form.
+    final String textValue = request.getParameter("text-input");
+
+    // Print the value so you can see it in the server logs.
+    System.out.println("You submitted: " + textValue);
```

Comment:
nit: You can remove these comments. The don't add much value since it is clear that you print the value, since it is a printf.
The reason for not adding unnecessary comments is that you might update the code and forget to update the comment. and the code and the comment will become out of sync.